transport.go

```diff
@@ -397,6 +404,11 @@ func (t *HTTPTransport) Configure(options ClientOptions) {
 
 // SendEvent assembles a new packet out of Event and sends it to remote server.
 func (t *HTTPTransport) SendEvent(event *Event) {
+	t.SendEventWithContext(context.Background(), event)
+}
+
+// SendEvent assembles a new packet out of Event and sends it to remote server.
+func (t *HTTPTransport) SendEventWithContext(ctx context.Context, event *Event) {
```

Comment:
Can we fix the grammar here please, even if it's just a copy of the old mistakes?